save-backend/src/test/kotlin/org/cqfn/save/backend/service/PermissionServiceTest.kt

```diff
@@ -79,7 +79,7 @@ class PermissionServiceTest {
     }
 
     private fun PermissionService.getRole(userName: String, projectName: String, organizationName: String): Mono<Role> =
-             findUserAndProject(userName, organizationName, projectName).map { (user, project) ->
+            findUserAndProject(userName, organizationName, projectName).map { (user, project) ->
```

Comment:
Very unexpected fact: in https://github.com/analysis-dev/save-cloud/runs/5646708887?check_suite_focus=true diktat_check actually failed, but gradle-cache-action failed silently and the check has been green. That's why we now have code style violation in master.